lib/cartopy/mpl/geoaxes.py

```diff
@@ -427,10 +427,45 @@ def draw(self, renderer=None, inframe=False):
                 self.imshow(img, extent=extent, origin=origin,
                             transform=factory.crs, *args[1:], **kwargs)
         self._done_img_factory = True
-
+        self._cachedRenderer = renderer
         return matplotlib.axes.Axes.draw(self, renderer=renderer,
                                          inframe=inframe)
 
+    def _update_title_position(self, renderer):
+        """
+        Update the title position based on the bounding box enclosing
+        all the ticklabels and x-axis spine and xlabel...
+
```

Comment:
Why the dots here?  Is this docstring unfinished?

Reply:
This docstring as taken from matplotlib:
https://github.com/matplotlib/matplotlib/blob/529410fa0c9379cd60b1e4d8fdd0478bb2f7e892/lib/matplotlib/axes/_base.py#L2468
Should I remove it or copy the __doc__ attribute?